### PR TITLE
Make distributable post statuses filterable

### PR DESCRIPTION
**Pull request by barryceelen**
The changes in #110 resulted in only allowing published posts to be distributed, with a possible exception for drafts via a filter.

This pull request adds a filterable `distributable_post_statuses()` utility function similar to `distributable_post_types()`. It is used in the `syndicatable()` function and replaces the previously added `dt_drafts_can_be_distributed` filter.

### CODE REVIEW

File: includes/push-ui.php [hunk 1]
```diff
@@ -20,7 +20,6 @@ function setup() {
 	);
 }
 
-
```
Comment (adamsilverstein) [inline includes/push-ui.php]: unrelated change :)

Comment (barryceelen) [reply includes/push-ui.php]: Cleanin' up dem whitespaces 😄
